packages/interfaces/src/IAppliance.js

```diff
@@ -58,17 +58,21 @@ class IAppliance {
 	}
 
 	/**
-	 * Prepares the Appliance to process data.
+	 * Prepares the Appliance to process data and starts data processing.
+	 *
+	 * @return {Boolean} Whether the appliance successfully started.
 	 */
-	setup = async () => {
-		throw new NotImplementedError('setup')
+	start = async () => {
+		throw new NotImplementedError('start')
 	}
 
 	/**
-	 * Cleans up after processing is complete.
+	 * Stops the appliance from processing additional data and cleans up the appliance.
+	 *
```

Comment:
It's implied by "processing additional data", but it may be good to explicitly mention what happens to existing data already queued for processing, or what the expectation is here.

If this isn't the right place to describe that behavior, or if that behavior is made explicit by some other piece of architecture, than never mind!

Reply:
Good catch.